src/smartpeak/source/core/SessionHandler.cpp

```diff
@@ -1609,6 +1609,10 @@ namespace SmartPeak
       }
     }
   }
+  void SessionHandler::resetChromatogramRange()
```

Comment:
Could you add in a unit test for `resetChromatogramRange` and `resetSpectrumRange`?

Reply:
sure, will do